source/audio.py

```diff
@@ -0,0 +1,204 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2015-2021 NV Access Limited
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+import atexit
+from comInterfaces.coreAudio.constants import (
+	CLSID_MMDeviceEnumerator,
+	EDataFlow,
+	ERole,
+)
+import comInterfaces.coreAudio.audioclient as audioclient
+import comInterfaces.coreAudio.audiopolicy as audiopolicy
+import comInterfaces.coreAudio.mmdeviceapi as mmdeviceapi
+import comtypes
+import config
+from enum import IntEnum, unique
+import globalVars
+import json
+from logHandler import log
+import nvwave
+from typing import Callable
+import ui
+from utils.displayString import DisplayStringIntEnum
+import _ctypes
+
+VolumeTupleT = tuple[float, float]
+
+
+@unique
+class SoundSplitState(DisplayStringIntEnum):

```

Comment:
I think I'd prefer having two options, one to select where NVDA's sound is played and one for the application sounds. That feels more intuitive to me.

Reply:
The original and main use case of this PR is sound split, so I would like users to be able to turn it on with a single keystroke instead of having to press two separate keystrokes one for NVDA and one for other apps.

Reply:
I agree that the list of possible sound split modes is a bit long, and if we can find something clearer, why not.

But, as @mltony writes, @LeonarddeR's suggestion of one option for NVDA sound and one for the sound of other apps would completely defeat the main use case of this feature.

Reply:
Re the concern with the sound split modes being too long:
If we really want 2 options, we may divide the size of the list by 2 and use a checkbox to switch left and right.

Reply:
AH yes, I see. Having two combo boxes would also allow a user to set NVDA right, apps right, which makes no sense at all.